devel-common/pyproject.toml

```diff
@@ -55,6 +55,7 @@ dependencies = [
 "debuggers" = [
     "ipdb>=0.13.13",
     "pdbr>=0.8.9",
+    "keyrings.alt>=5.0.2",
```

Comment:
@potiuk, should I include this under `debuggers`?

Reply:
I'd say - separate optional dependency - named keyring

Reply:
Actually .. hmm. It's ilikely better to add it to "dev" dependency group in `airflow-ctl` - that's where it belongs

Reply:
More explanation: The "dev" dependency group is always installed when `uv sync` is run. So, if you add  it in "dev" dependency group in airflow-ctl's `pyproject.toml`, you will get it installed:

```
cd airflow-ctl
uv sync
```

Reply:
Created similar to devel-common and other distributions group structure, thanks Jarek!